pages/docs/reference/whatsnew14.md

```diff
@@ -729,4 +729,18 @@ kotlin {
     }
 }
 ```
-</div>
\ No newline at end of file
+</div>
+
+## kotlinx.serialization 1.0.0
+
+With Kotlin 1.4, we ship the first stable version of [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization) - 1.0.0.
+For now, we declare the JSON serialization library (`kotlinx-serialization-runtime`) stable. Libraries for other serialization formats still remain experimental.
```

Comment:
Some significant parts of the core still would be marked as `@ExpreimentalSerializationApi`. However, that parts are connected with advanced usages only, like schema introspecting or writing your own serialization format.